LICENSE

```diff
@@ -0,0 +1,99 @@
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             BabylonLabs, Ltd.
+
+Licensed Work:        finality-provider
```

Comment:
This is not the licensed work.

Reply:
Should I remove this?

Reply:
It should include it, please follow exactly the same format as with other repositories and modify only the parameters. The licensed work should be `Babylon GitHub Workflows`